contrib/pg_tde/src/catalog/tde_keyring.c

```diff
@@ -184,15 +191,28 @@ cleanup_key_provider_info(Oid databaseId)
 	PathNameDeleteTemporaryFile(kp_info_path, false);
 }
 
+static Datum
+			pg_tde_change_key_provider_c(PG_FUNCTION_ARGS, Oid dbOid, int shift);
```

Comment:
Odd linebreak here.

Reply:
And sholdn't all function prototypes be collected at the top of the C file?

Reply:
These comments apply to all 5 functions.

Reply:
> And sholdn't all function prototypes be collected at the top of the C file

Fixed. This is usually just an afterthought on my part after the compiler complains, it's really annoying that in C, even static functions need a separate declaration, there's not much point to these.